arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -1289,6 +1288,38 @@ def rule_gfm_1_4_4(
                 )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_4_8(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.4.8] Correct the priority attribute value so that it is less than 10.
+    """
+
+    for modelDocument in val.modelXbrl.urlDocs.values():
+        if pluginData.isStandardTaxonomyUrl(modelDocument.uri, val.modelXbrl):
+            continue
+        rootElt = modelDocument.xmlRootElement
+        ns = {'xlink': 'http://www.w3.org/1999/xlink'}
+        for elt in rootElt.xpath('//*[@xlink:type="arc"]', namespaces=ns):
+            if elt.get("priority") is not None:
+                priority = elt.get("priority")
+                if int(priority) >= 10:
```

Comment:
We should guard against type/value errors on the int conversion.

nit: it's also slightly more efficient to filter in xpath/c code for elements with a priority attribute.


```suggestion
        for elt in rootElt.xpath('//*[@xlink:type="arc"][@priority]', namespaces=ns):
            priority = elt.get("priority")
            with contextlib.suppress(ValueError, TypeError):
                if int(priority) >= 10:
```